### PR TITLE
Add styling to the site

### LINKED ISSUE #6: Add styling to the site
We need to add some design to the site generated to make it look more beautiful.

coala uses materialize css for designing the webpages. You can find some inspiration in the other repos which render html pages.

### PR DESCRIPTION
**Pull request by siddhpant**
Closes #6.

### CODE REVIEW

File: templates/home.html.jinja2 [hunk 1]
```diff
@@ -1,19 +1,53 @@
...
+          <div class="col s6 m4 l2" style="display: inline-block">
+            <div class="card hoverable transparent card-small">
+              <div class="card-image">
+                <a href="{% url 'languages', language %}">
+                  <img src="assets/logos/{{ m.language_id(language) }}.svg" class="lang-logo">
+                </a>
+              </div>
+            </div>
+          </div>
+        {% endfor %}
```
Comment (jayvdb) [inline templates/home.html.jinja2]: Unlabeled control end tag

*Origin: Jinja2Bear, Section: `jinja2`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwbn_drex/templates/home.html.jinja2
+++ b/tmp/tmpwbn_drex/templates/home.html.jinja2
@@ -43,7 +43,7 @@
               </div>
             </div>
           </div>
-        {% endfor %}
+        {% endfor %}{# for language in languages #}
       </div>
     </div>
   </div>
```

Comment (jayvdb) [inline templates/home.html.jinja2]: Unlabeled control end tag

*Origin: Jinja2Bear, Section: `jinja2`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwbn_drex/templates/home.html.jinja2
+++ b/tmp/tmpwbn_drex/templates/home.html.jinja2
@@ -43,7 +43,7 @@
               </div>
             </div>
           </div>
-        {% endfor %}
+        {% endfor %}{# for language in languages #}
       </div>
     </div>
   </div>
```

Comment (jayvdb) [inline templates/home.html.jinja2]: Unlabeled control end tag

*Origin: Jinja2Bear, Section: `jinja2`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwbn_drex/templates/home.html.jinja2
+++ b/tmp/tmpwbn_drex/templates/home.html.jinja2
@@ -43,7 +43,7 @@
               </div>
             </div>
           </div>
-        {% endfor %}
+        {% endfor %}{# for language in languages #}
       </div>
     </div>
   </div>
```

Comment (siddhpant) [reply templates/home.html.jinja2]: Oh I have to squash all  for this

Comment (jayvdb) [inline templates/home.html.jinja2]: Unlabeled control end tag

*Origin: Jinja2Bear, Section: `jinja2`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwbn_drex/templates/home.html.jinja2
+++ b/tmp/tmpwbn_drex/templates/home.html.jinja2
@@ -43,7 +43,7 @@
               </div>
             </div>
           </div>
-        {% endfor %}
+        {% endfor %}{# for language in languages #}
       </div>
     </div>
   </div>
```

Comment (siddhpant) [reply templates/home.html.jinja2]: how

**Comment by jayvdb** [comment]
https://deploy-preview-98--coast.netlify.com/ - very nice!

**Comment by siddhpant** [comment]
wherever was necessary

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/98/commits/d781c039702235b6c81a43dda7d9cb88f8a59a74

**Pull request merged**